src/Chip/ChipAdornment.tsx

```diff
@@ -0,0 +1,53 @@
+import clsx from "clsx";
+import React from "react";
+
+import { makeStyles } from "../theme";
+
+const useStyles = makeStyles(
+  (theme) => ({
+    adornment: {
+      color: theme.palette.primary.contrastText,
+      opacity: "60%",
```

Comment:
```suggestion
      opacity: 0.6,
```
https://caniuse.com/mdn-css_properties_opacity_percentages